clients/drcachesim/tests/burst_replaceall.cpp

```diff
@@ -42,11 +42,24 @@
 #include "drmemtrace/drmemtrace.h"
 #include "drvector.h"
 #include "../../../suite/tests/client_tools.h"
+#include "../../../suite/tests/condvar.h"
 #include <assert.h>
 #include <iostream>
 #include <math.h>
 #include <stdlib.h>
 #include <string.h>
+#include <set>
+#ifndef WINDOWS // XXX i#2040: Static client limitations
+#    ifdef WINDOWS
+#        include <process.h>
+#    endif
+#endif
+
+static void *finished;
```

Comment:
`finished` can also be conditionally defined for !WINDOWS.

Reply:
Done.